generation/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -181,23 +157,23 @@
     "files = [\n",
     "    {\n",
     "        \"path\": \"models/autoencoder_epoch273.pt\",\n",
-    "        \"url\": \"https://drive.google.com/file/d/1jQefG0yJPzSvTG5rIJVHNqDReBTvVmZ0/view?usp=drive_link\",\n",
+    "        \"url\": \"https://drive.google.com/file/d/1Ojw25lFO8QbHkxazdK4CgZTyp3GFNZGz/view?usp=sharing\",\n",
```

Comment:
Due to the security policy, these links might no longer work by the end of this month. We'll have to wait for @zephyrie to respond before putting them somewhere.

Reply:
@Nic-Ma

Reply:
It seems we may still use google drive? Should we continue merge this PR or wait for further decision?